pkg/client/clientcmd/overrides.go

```diff
@@ -109,8 +111,12 @@ func RecommendedConfigOverrideFlags(prefix string) ConfigOverrideFlags {
 
 // BindAuthInfoFlags is a convenience method to bind the specified flags to their associated variables
 func BindAuthInfoFlags(authInfo *clientcmdapi.AuthInfo, flags *pflag.FlagSet, flagNames AuthOverrideFlags) {
-	// TODO short flag names are impossible to prefix, decide whether to keep them or not
-	flags.StringVarP(&authInfo.AuthPath, flagNames.AuthPath, "a", "", "Path to the auth info file. If missing, prompt the user. Only used if using https.")
+	// TODO short flag names are impossible to prefix.  code gets cleaner if we remove them
+	if len(flagNames.AuthPathShort) > 0 {
+		flags.StringVarP(&authInfo.AuthPath, flagNames.AuthPath, flagNames.AuthPathShort, "", "Path to the auth info file. If missing, prompt the user. Only used if using https.")
```

Comment:
> This is bound to get out of sync at some point

If it's still in there tomorrow, I'll update to use a common variable.



> call flags.StringVarP and pass it a "" as short option it will simply keep no short option for it

I don't see that in the docs, though I agree that the code actually does that.  It feels like that's coincidence, not API.  I don't know that I'd feel comfortable doing that.